code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -145,6 +145,20 @@
 		else
 			user << "<span class='warning'>You need one rod to make a wired rod!</span>"
 			return
+	else if(istype(I, /obj/item/stack/sheet/metal))
+		var/obj/item/weapon/restraints/legcuffs/bola/S = new /obj/item/weapon/restraints/legcuffs/bola
+		var/obj/item/stack/sheet/metal/M = I
+		if(M.amount < 6)
+			user << "<span class='warning'>You need at least six metal sheets to make good enough weights!</span>"
+			return
+		user << "<span class='notice'>You begin to apply [I] the [src]...</span>"
+		if(do_after(user, 35, target = src))

```

Comment:
Here is where you'd want to make the bolas--after the do after has passed---you already have checks for the metal amount. above.